conformance/operator_test.go

```
@@ -641,16 +644,17 @@ var _ = Describe("operator", func() {
 	})
 })
 
-func getMacFrom(interfaceOutput string) string {
-	interfaceOutput = strings.TrimSpace(interfaceOutput)
-	lines := strings.Split(interfaceOutput, "\n")
-	Expect(len(lines)).To(Equal(6))
-
-	macConfig := strings.TrimSpace(lines[1])
-	macConfigSplit := strings.Split(macConfig, " ")
-	Expect(len(macConfigSplit)).To(Equal(4))
+// Find the VF id
+// This function goes over the vf interfaces name and report if one of them is not in the host network namespace
+func findVF(ifaceName string, numVfs int, podObj *corev1.Pod) int {
```

Can you have this return an error instead of returning -1?
Also, can this be findPodVFInHost(pfName, ..) ?
And can you make the comment more go idiomatic (something like findPodVFInHost goes over...), and expand it a bit? This is a bit hacky and hard to understand otherwise